app/helpers/application_helper.rb

```diff
@@ -283,6 +283,9 @@ def db_to_controller(db, action = "show")
       action = "cim_base_storage_extents"
     when "ConditionSet"
       controller = "condition"
+    when "EmsInfra"
+      controller = "ems_infra"
+      action = nil
```

Comment:
@mzazrivec How come I do not see identical code for `EmsCloud` since that is RESTful too?(in fact, the first controller with restful changes)